Bottom: 296f1c2066c8f8c7729ae3381286d6e689b1d40c
Top:    64506f3730b3d07ce4b526ca8cca62d781989e44
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-29 23:41:01 +0100

Refresh of draw-c-fold-drawlayercommon-in

---

diff --git a/src/draw.c b/src/draw.c
index 83080b3..ccdc46d 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1072,18 +1072,15 @@ text_callback (const BoxType * b, void *cl)
   return 1;
 }
 
-/* ---------------------------------------------------------------------------
- * draws one non-copper layer
- */
 void
-DrawLayerCommon (LayerTypePtr Layer, const BoxType * screen, bool clear_pins)
+DrawLayer (LayerTypePtr Layer, const BoxType * screen)
 {
   struct poly_info info = {screen, Layer};
 
   /* print the non-clearing polys */
   r_search (Layer->polygon_tree, screen, NULL, poly_callback, &info);
 
-  if (clear_pins && TEST_FLAG (CHECKPLANESFLAG, PCB))
+  if (TEST_FLAG (CHECKPLANESFLAG, PCB))
     return;
 
   /* draw all visible lines this layer */
@@ -1109,11 +1106,6 @@ DrawLayerCommon (LayerTypePtr Layer, const BoxType * screen, bool clear_pins)
 		      PCB->MaxWidth, PCB->MaxHeight);
     }
 }
-
-void
-DrawLayer (LayerTypePtr Layer, const BoxType * screen)
-{
-  DrawLayerCommon (Layer, screen, false);
 }
 
 /* ---------------------------------------------------------------------------
@@ -1137,7 +1129,7 @@ DrawLayerGroup (int group, const BoxType *drawn_area)
           strcmp (Layer->Name, "route") == 0)
         rv = 0;
       if (layernum < max_copper_layer && Layer->On)
-        DrawLayerCommon (Layer, drawn_area, true);
+        DrawLayer (Layer, drawn_area);
     }
   if (n_entries > 1)
     rv = 1;
